.gitignore

```diff
@@ -56,3 +56,4 @@ typings/
 
 # dotenv environment variables file
 .env
+env.json
```

Comment:
good call. for safety I always put the env.json file one level up.  that also enables sharing of the file between our different services.